backend_tv/lifter.cpp

```diff
@@ -2046,7 +2046,7 @@ UMLALv8i16_v4i32
     auto i32 = getIntTy(32);
     auto i64 = getIntTy(64);
 
-    if (false) {
```

Comment:
what happened here? did your previous pull request change this?

Reply:
Yes. Sorry, i forgot I changed this in my last commit.

Reply:
ok please be super careful to not change stuff like that!